apps/bfd-pipeline/bfd-pipeline-app/pom.xml

```diff
@@ -142,6 +142,12 @@
             <artifactId>awaitility</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <!-- Used to write fluent assertions in unit tests. -->
+            <groupId>org.assertj</groupId>
+            <artifactId>assertj-core</artifactId>
+            <scope>test</scope>
+        </dependency>
```

Comment:
@StewGoin 👋 We're attempting to introduce a new test-scoped dependency here:
* actively maintained: https://github.com/assertj/assertj
* widely used: https://mvnrepository.com/search?q=assertj-core
* recent activity on SO: https://stackoverflow.com/questions/tagged/assertj